hydromt/vector.py

```diff
@@ -575,15 +581,17 @@ def to_gdf(self, reducer=None):
 
         Arguments
         ---------
-        reducer : callable, optional
+        reducer : callable, str, optional
             method by which multidimensional data is reduced to 1 dimensional
-            e.g. numpy.mean
+            e.g. numpy.mean. If str (e.g. mean), will call the numpy method is found.
 
         Returns
         -------
         gdf: geopandas.GeoDataFrame
             GeoDataFrame
         """
+        if reducer is not None and isinstance(reducer, str):
```

Comment:
`reducer is not None` is not needed here as `None` is not an instance of `str`, but it's a small nitpick. Up do you if you want to change it.